Rakefile

```diff
@@ -35,7 +35,10 @@ begin
   require 'rspec/core/rake_task'
 
   desc 'Run ChefSpec examples'
-  RSpec::Core::RakeTask.new(:spec)
+  RSpec::Core::RakeTask.new(:spec) do |t|
```

Comment:
revert this, i will rip the spec apart once the rest of the rewrite is done
Unless you are volunteering :)